src/synapse/synapse_handler.rs

```diff
@@ -24,24 +24,30 @@ fn build_room_local_alias(acting_user: &str, second_user: &str) -> String {
     addresses.join("+")
 }
 
-/// When accepting a friend request, then the room invitation needs to be accepted too. Check out [here](https://spec.matrix.org/v1.3/client-server-api/#room-membership) to find out more about room membership and permissions based on the state in which a user may be.
+/// When interacting with a friend, then you need to join the room if you hadn't joined it yet.
+/// Check out [here](https://spec.matrix.org/v1.3/client-server-api/#room-membership) to find out more about room membership and permissions based on the state in which a user may be.
 pub async fn accept_room_invitation<'a>(
     token: &str,
     room_id: &str,
     room_event: FriendshipEvent,
     room_message_body: Option<&str>,
```

Comment:
What is it for the `room_message_body` in this context?

Reply:
nice catch!